src/pymedphys/dcm/_level1/dcmdose.py

```diff
@@ -508,7 +431,7 @@ def extract_dicom_patient_coords(dcm):
     else:
         z = np.flip(-position[2] + np.array(dcm.GridFrameOffsetVector))
 
-    return Coords(x, y, z)
```

Comment:
Not much to be gained from the nametuple, I don't think.

Reply:
Happy to lean on your judgement here.

Reply:
Also, as a protocol for resolving discussions, for me this is resolved, bit if a click resolved I fear you might miss the comment. So maybe, we only click resolve if the other person was the last to comment? What do you think?

Reply:
Seems reasonable. Do you want to try resolving one of these just to see how it manifests?